sympy/functions/elementary/tests/test_hyperbolic.py

```diff
@@ -678,6 +681,11 @@ def test_acosh():
     assert acosh(cosh(cosh(1) + I)) == cosh(1) + I
     assert acosh(1, evaluate=False).is_zero is True
 
+    # Reality
+    assert acosh(2).is_real
+    assert not acosh(0.5).is_real
+    assert (acosh(2) - oo) == -oo
```

Comment:
It is better to assert that True, False etc are actually returned explicitly:
```python
assert acosh(2).is_real is True
```
We don't want `is_real` to return any other "truthy" object. Same goes for asserting with `not` which would pass for either False or None.

Note that `acosh` with a Float evaluates:
```
In [7]: acosh(0.5)
Out[7]: 1.0471975511966⋅ⅈ
```
It would be better to test with a Rational:
```python
In [10]: acosh(S(1)/5)
Out[10]: acosh(1/5)

In [11]: acosh(S(1)/5).is_real
Out[11]: False
```